.github/ISSUE_TEMPLATE.md

```diff
@@ -1,8 +1,19 @@
-## **Problem:**
-
-
-## **Proposed Solution:**
+### Problem:
+A short description of what the problem is and why we need to fix it. Add reproduction steps if necessary.
+### Solution:
+A description of the possible solution in terms of S2N architecture. Highlight and explain any potentially controversial design decisions taken.
 
+**Does this change what S2N sends over the wire?** If yes, explain.
+**Does this change any public APIs?** If yes, explain.
```

Comment:
If there are changes/additions to public APIs we must make sure that the documentation/Usage Guide is updated.

Reply:
Does this make sense as a separate question, or just as part of the question about changes to public APIs?